### ISSUE TITLE
Usage as CV

### ISSUE THREAD

**Issue by lauhon** [netlify/million-devs#70]
Hi, 

I would love to use the code and graphics as template to create a CV.
I created [this fork](https://github.com/lauhon/my-page). Is this ok, or would netlify view this as theft and possibly sue me if I use it as my CV?

I know this isnt really an issue, but i wasnt sure where to reach out. Thanks for any response :)

**Comment by sdras** [comment]
Hey @lauhon,

Thanks for asking. You're correct that this is open source and thus fair and legal to use.

However, usage for a CV implies you made it yourself, which while legally fine is morally ambigious. Without being clear that you didn't write this code- I did- you're taking credit for someone else's work, which isn't really the point of open source. What if you get a job where they expect you to build something like this from scratch in a different way? Or what if they saw this original site before and reject your application?

My suggestion would be to learn from it, modify it, and make it your own. If you're not working with a designer like I did, there are a lot of pre-built vectors to buy or even for free to make things out of, such as freepic, ui8.net, shutterstock, and iphoto.

Thanks and good luck!
